host/src/pages/Dashboard/Reservation/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { BreadcrumbItem, Breadcrumbs, Tab, Tabs } from "@nextui-org/react";
+import { Link } from "react-router-dom";
+import { DashboardRoutes } from "@/types/routes";
+import IncomingReservation from "@/components/Reservations/incoming-reservation";
+import ConfirmedReservation from "@/components/Reservations/confirmed-reservation";
+import RejectedReservation from "@/components/Reservations/rejected-reservation";
+
+export default function Reservation(): JSX.Element {
+  return (
+    <div className="py-6 space-y-6">
+      <Breadcrumbs>
```

Comment:
Do we need still need Breadcrumbs when the Home route is just by the sidebar and `/reservations` route is not a nested route

Reply:
ohh i think that wont be necessary, would just take it off then